src/Orchard.Web/Modules/Orchard.DynamicForms/Drivers/RadioButtonElementDriver.cs

```diff
@@ -34,8 +34,14 @@ public RadioButtonElementDriver(IFormsBasedElementServices formsServices, IToken
                         Name: "Value",
                         Title: "Value",
                         Classes: new[] { "text", "medium", "tokenized" },
-                        Description: T("The value of this radio button.")));
-
+                        Description: T("The value of this radio button.")),
+                    _DefaultValue:
```

Comment:
I don't see what's wrong with formatting. _DefaultValue var is at the same level of indentation of its sibling var Description. 
Do you mean '_' char? I saw other elements using a '_' char before the name of the variable. Do I have to rename Description to _Description or _DefaultValue to DefaultValue ?